copado-function/app/Commit.js

```diff
@@ -581,6 +594,21 @@ class Copado {
         );
     }
 
+    /**
+     * Deletes the remote feature branch
+     *
+     * @param {string} featureBranch branch that is going to be deleted
+     * @returns {void}
+     */
+    static deleteBranch(featureBranch) {
+        // delete feature branch on origin code in here
+        Util.execCommand(
+            'Deleting branch ' + featureBranch,
+            [`git push origin --delete ${featureBranch}`],
```

Comment:
looking at https://stackoverflow.com/a/2003515/818732 it seems you are only deleting the remote branch but not the local version - which will make recreating it potentially fail?

can you please test if there suggestion causes issues?

```suggestion
            [`git push origin --delete ${featureBranch}`, `git branch --delete --force ${featureBranch}`],
```

git push -d <remote_name> <branchname>
git branch -d <branchname>